primitives/tf_encrypted/primitives/sodium/python/easy_box_test.py

```diff
@@ -55,20 +82,15 @@ def test_open_uint8(self):
         assert plaintext_recovered.shape == plaintext.shape
         np.testing.assert_equal(plaintext_recovered, np.array([[1, 2], [3, 4]]))
 
-    def test_open_float(self):
-        pk_s, sk_s = easy_box.gen_keypair()
-        pk_r, sk_r = easy_box.gen_keypair()
 
-        plaintext = tf.constant([1, 2, 3, 4], shape=(2, 2), dtype=tf.float32)
+def run_op(op, mode):
```

Comment:
I've generalized this a tiny bit, see my commit

Reply:
Nice! I didn't know `contextlib.suppress()`. Looks great!